sdk/core/azure-core-http-jdk-httpclient/src/main/java/com/azure/core/http/jdk/httpclient/JdkHttpClientProvider.java

```diff
@@ -1,14 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.core.http.jdk.httpclient.implementation;
+package com.azure.core.http.jdk.httpclient;
 
 import com.azure.core.http.HttpClient;
 import com.azure.core.http.HttpClientProvider;
-import com.azure.core.http.jdk.httpclient.JdkAsyncHttpClientBuilder;
 
 /**
- * A {@link HttpClientProvider} that provides an implementation of HttpClient based on native JDK HttpClient.
+ * An {@link HttpClientProvider} that provides an implementation of HttpClient based on native JDK HttpClient.
```

Comment:
Can we also add documentation to mention that this is for JDK 11 and above only?